vng_api_common/permissions.py

```diff
@@ -78,35 +80,50 @@ def has_object_permission(self, request: Request, view, obj) -> bool:
             return request.jwt_payload['client_id'] == obj.client_id
 
 
-class AuthScopesRequired(permissions.BasePermission):
+class BaseAuthRequired(permissions.BasePermission):
     """
     Look at the scopes required for the current action
     and check that they are present in the AC for this client
     """
+    permission_fields = None
+    get_obj = None
+    obj_path = None
 
-    def get_zaaktype(self, obj):
-        return None
+    def _get_obj(self, view, request):
+        if not isinstance(self.get_obj, str):
+            raise TypeError("'get_obj' must be set to a string, representing a view method name")
 
-    def get_vertrouwelijkheidaanduiding(self, obj):
-        return None
+        method = getattr(view, self.get_obj)
+        return method()
 
-    def get_zaaktype_from_request(self, request):
-        return None
+    def _get_obj_from_path(self, obj):
+        if not isinstance(self.obj_path, str):
+            raise TypeError("'obj_path' must be a python dotted path to the main object FK")
 
-    def get_vertrouwelijkheidaanduiding_from_request(self, request):
-        return None
+        bits = self.obj_path.split('.')
+        for bit in bits:
+            obj = getattr(obj, bit)
+        return obj
+
+    def _extract_field_value(self, main_obj, field):
+        return getattr(main_obj, field)
+
+    def get_permission_fields(self):
+        return self.permission_fields or {}
```

Comment:
shouldn't this be a list/tuple instead of a dict?

I'd opt for setting `permissions_fields = ()` in as well instead of `None`

Reply:
Thanks for catching a bug. Yeah, I set `permissions_fields = ()` and removed `get_permission_fields` method